src/pybind/mgr/volumes/module.py

```diff
@@ -591,6 +590,25 @@ def config_notify(self):
                     elif opt['name'] == "snapshot_clone_no_wait":
                         self.vc.cloner.reconfigure_reject_clones(self.snapshot_clone_no_wait)
 
+    def _is_valid_name(self, name):
+        if not name:  # for None ... e.g. we're okay when group_name is None
+            return True
+        sn = set(name)
+        # leading dot creates hidden dirs for groups and subvolumes
+        # that doesn't seem like a sensible thing to have
+        if name.startswith('.') or sn == set('.') or sn == set('/') or sn == set('./'):
+            return False
+        return True
+
+    def _is_valid_names(self, vol_name, group_name, sub_name):
```

Comment:
Since often call to this method passes `None` for last 2 arguments it's better to define it as the default value for these arguments. This will make passing `None` repeatedly in every call redundant. So, calls will look shorter and cleaner.

```suggestion
    def _is_valid_names(self, vol_name, group_name=None, sub_name=None):
```

Reply:
renamed to `_are_names_valid(...)` and introduced default values for `group_name` and `sub_name`